Makefile

```diff
@@ -15,8 +15,6 @@
 # Various versions - Minishift, default OpenShift, default B2D ISO
 MINISHIFT_VERSION = 1.3.1
 OPENSHIFT_VERSION = v1.5.1
-B2D_ISO_VERSION = v1.0.2
-CENTOS_ISO_VERSION = v1.1.0
```

Comment:
> But I am still not sure how we want to handle the versions of ISOs we put in docs.

I don't think we really need the concrete ISO version in the docs. ATM, it is only used once as an example. We don't need this dynamic really.